composer.json

```diff
@@ -2,7 +2,8 @@
     "name": "compositephp/performance",
     "autoload": {
         "psr-4": {
-            "App\\": "src/"
+            "App\\": "src/",
+            "PHPFUIORM\\": "src/PHPFUIORM/"
```

Comment:
Is it possible to avoid new namespace definition?

Reply:
Yes, fixed in latest commit.  I did not realize you were using more of a library type style for the benchmarks.